src/Provider/Doctrine/FixtureFactory.php

```diff
@@ -100,9 +104,8 @@ public function get($name, array $fieldOverrides = array())
         if (isset($config['afterCreate'])) {
             $config['afterCreate']($ent, $fieldValues);
         }
-        
-        
-        if ($this->persist) {
+
+        if ($this->persist && (!property_exists($entityMetadata, 'isEmbeddedClass') || false === $entityMetadata->isEmbeddedClass)) {
```

Comment:
Not sure if this is too verbose, but the field may not yet exist, so I better check for its existence.

For reference, see 

* http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/changelog/migration_2_5.html#embeddable-objects
* https://github.com/doctrine/doctrine2/pull/835

Reply:
sure, that's ok